SalesforceCommerceCloud/pwa-kit
# Update PWA Kit to work with the new SLAS channel_id & shopper context siteId change (@W-16253008@)

# Description



The PR consumes the changes in SLAS channel_id and shopper context siteId by bumping commerce-sdk-isomorphic version to the latest.



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Bump to commerce-sdk-isomorphic@3.0.0

# How to Test-Drive This PR

1. Checkout this branch `W-16253008-update-pwa-kit-with-channel_id-and-siteId`.
2. Enable and [configure the Private client](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/use-a-slas-private-client.html#configure-the-pwa-kit):
    2.1. `template-retail-react-app/config/default.js` 
    ```
    clientId: '8c7ad811-647f-48ae-9e51-992bd7ea33b3',
    ```
 
    2.2 `template-retail-react-app/app/ssr.js`
    ```
    useSLASPrivateClient: true
    ```
  
    2.3 `template-retail-react-app/app/components/_app-config/index.jsx`
    ```
    enablePWAKitPrivateClient={true}
    ```

3. Start the retail-react-app with the secret private client: `PWA_KIT_SLAS_CLIENT_SECRET=<value> npm start`.
4. Verify in the browser network tab that all SLAS requests get 200 responses.


# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
